client/components/form/Description.tsx

```diff
@@ -1,5 +1,5 @@
 interface DescriptionProps {
-  id: string;
+  id?: string;
```

Comment:
Both of these props are highly important to this component, I think they should both be required